src/axom/sina/docs/sphinx/documents.rst

```diff
@@ -0,0 +1,192 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _documents-label:
+
+=========
+Documents
+=========
+
+Sina ``Document`` objects are a way to represent the top-level object of a
+JSON file that conforms to the Sina schema. When serialized, these documents
+can be ingested into a Sina database and used with the Sina tool.
+
+``Document`` objects follow a very basic JSON layout consisting of two entries:
+"records" and "relationships". Each of these entries will store a list of their
+respective objects. An example of an empty document is shown below:
+
+.. code:: json
+
+    {
+        "records": [],
+        "relationships": []
+    }
+
+The "records" list can contain ``Record`` objects and their inheritying types,
+such as ``Run`` objects. The "relationships" list can contain ``Relationship``
+objects. For more information on these objects, see `Records <./records>`_
+and `Relationships <./relationships>`_.
+
+--------------------
+Assembling Documents
+--------------------
+
+``Document`` objects can be assembled programatically. To accomplish this:
+
+1. Create a new instance of the ``Document`` class
+2. Create a ``Record``
+3. Add the instance of the ``Record`` with the ``add`` method
+
+On the `Sina C++ User Guide <./index>`_ page, you can see an example of this
+process. Below we will expand on this example to add a ``Relationship``:
+
+.. code:: cpp
+
+    #include "axom/sina.hpp"
```

Comment:
Can this be put into an example so that it will be ensure it continues to compile?